src/node_worker.cc

```diff
@@ -132,7 +132,16 @@ class WorkerThreadData {
  public:
   explicit WorkerThreadData(Worker* w)
     : w_(w) {
-    CHECK_EQ(uv_loop_init(&loop_), 0);
+    int ret = uv_loop_init(&loop_);
+    if (ret != 0) {
+      char err_buf[128];
+      uv_err_name_r(ret, err_buf, sizeof(err_buf));
+      std::string error_str = SPrintF("ERR_WORKER_INIT_FAILED: %s", err_buf);
+      w->custom_error_ = error_str;
+      w->loop_init_failed_ = true;
+      w->stopped_ = true;
```

Comment:
I don’t think setting this has any effect (and it’s also not set in the other early return from this method)

Reply:
@addaleax 
> don’t think setting this has any effect
```
bash-4.2$ ./node bar.js 10000
./node[20714]: ../src/node_worker.cc:451:virtual node::worker::Worker::~Worker(): Assertion `stopped_' failed.
 1: 0xa06de0 node::Abort() [./node]
 2: 0xa06e67  [./node]
 3: 0xaa30d2 node::worker::Worker::~Worker() [./node]
 4: 0xaa3222  [./node]
 5: 0x9a47cc node::Environment::RunAndClearNativeImmediates(bool) [./node]
 6: 0x133a841  [./node]
 7: 0x134d168  [./node]
 8: 0x133b1cb uv_run [./node]
 9: 0xa455d3 node::NodeMainInstance::Run() [./node]
10: 0x9d8187 node::Start(int, char**) [./node]
11: 0x7f8d5ec68545 __libc_start_main [/lib64/libc.so.6]
12: 0x973d55  [./node]
```
this is what I get without that.

> (and it’s also not set in the other early return from this method)

Set the flag in the other early return also.